requirements_all.txt

```diff
@@ -1150,7 +1150,7 @@ plumlightpad==0.0.11
 pmsensor==0.4
 
 # homeassistant.components.pocketcasts
-pocketcasts==0.1
+pycketcasts==1.0.0
```

Comment:
Please run `python3 -m script.gen_requirements_all` to move this requirement to the correct place alphabetically.

Reply:
Done